src/swap/SwapScreen.tsx

```diff
@@ -108,8 +90,6 @@ export function SwapScreen({ route }: Props) {
   ).maxSlippagePercentage
   const parsedSlippagePercentage = new BigNumber(slippagePercentage).toFormat()
 
-  const useViemForSwap = getFeatureGate(StatsigFeatureGates.USE_VIEM_FOR_SWAP)
```

Comment:
So this PR also forces the new client to use `viem`.
I think this is fine, but I didn't expect this by reading the PR title or description.

Reply:
It's because it was initially based on my branch where I removed ContractKit, but then this PR got switched to the main branch when it was merged:

<img width="746" alt="Screenshot 2023-12-15 at 10 15 39" src="https://github.com/valora-inc/wallet/assets/57791/0e543b6e-7882-45dc-986c-a7f8cf43dcf5">

Reply:
yes sorry i should have mentioned this 😅 i decided not to bother with the contractkit flow because Jean's PR was about to be merged